updater/lib/dependabot/setup.rb

```diff
@@ -37,7 +37,31 @@
     pub
   )}x
 
-  config.processors += [ExceptionSanitizer]
+  config.before_send = lambda do |event, hint|
+    debugger
+    if !hint[:exception].nil?
+      ExceptionSanitizer.process_sentry(event, hint)
+
+      # TODO integrate these two...
+      # TODO copied from migration guide, I don't understand how this assignment works though?
+      # https://docs.sentry.io/platforms/ruby/migration/#exceptionraven_context
+      if exception = hint[:exception]
+        exception.raven_context.each do |key, value|
+          event.send("#{key}=", value)
+        end
+      end
+
+    else
+      event
+    end
+  end
+
+  # to send events synchronously like sentry-raven does, set it to 0
+  # config.background_worker_threads = 0
```

Comment:
note for myself: I'm guessing we should send synchronously, both for reduced risk by minimizing change and to ensure a job that's blowing up doesn't fire exception, then exit immediately before the thread pool has a chance to send the exception... I haven't looked into if there's a `flush()` style method, but IMO probably simplest to sidestep any complexity with a synchronous call... esp since the context where we call this is already a background job, no human is waiting on a low-latency response.